lib/tessel/deployment/rust.js

```diff
@@ -122,11 +122,17 @@ exportables.remoteRustCompilation = (opts) => {
         })
         // Reject on failure
         .on('error', function(postReqError) {
+          // Stop the spinner if we error
+          log.spinner.stop();
```

Comment:
I don't think this is needed because rejected or resolved commands from the cli stop the `spinner` -> https://github.com/tessel/t2-cli/blob/master/bin/tessel-2.js#L510-L534